app/scripts/directives/overviewService.js

```diff
@@ -48,14 +49,39 @@ angular.module('openshiftConsole')
           return isRecentDeployment(deployment, dc);
         };
 
+        $scope.visibleDeployments = function(deployments) {
+          return _.filter(deployments, $scope.isDeploymentVisible);
+        };
+
+        $scope.deploymentInProgress = function(deployments) {
+          return anyDeploymentIsInProgress(deployments);
+        };
+
+        $scope.isDeploymentLatest = function(deployment) {
```

Comment:
We have a `deploymentIsLatest` filter you should be able to use.

https://github.com/spadgett/origin-web-console/blob/e32941237fb8e9fbc76a932491f4d56d2455b31e/app/scripts/filters/resources.js#L778